apps/rehabstod/src/pages/Patient/components/PatientHeader.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import { Provider } from 'react-redux'
+import { createStore } from '@reduxjs/toolkit'
+import { fakePatient } from '../../../utils/fake/fakePatient'
+import { PatientHeader } from './PatientHeader'
+
+describe('PatientHeader', () => {
+  const patient = fakePatient()
+
+  function renderPatientHeader(showPersonalInformation: boolean) {
+    const initialState = { sickLeave: { showPersonalInformation } }
+    const store = createStore(() => initialState)
+    return render(
+      <Provider store={store}>
+        <MemoryRouter>
+          <PatientHeader patient={patient} />
+        </MemoryRouter>
+      </Provider>
+    )
+  }
+
+  beforeEach(() => {
+    patient.sjukfallList[0].intyg[0] = {
+      ...patient.sjukfallList[0].intyg[0],
+      start: '2023-04-25',
+      slut: '2023-05-01',
+      patient: {
+        ...patient.sjukfallList[0].intyg[0].patient,
+        alder: 30,
+        kon: 'Kvinna',
+        namn: 'Olle',
+        id: '123',
+      },
+    }
+  })
```

Comment:
We can make this a bit simpler, but i noticed an issue with how our faker package handles mergers.

```tsx
function renderPatientHeader(showPersonalInformation: boolean) {
  const patient = fakePatient({
    sjukfallList: [
      {
        intyg: [
          {
            start: '2023-04-25',
            slut: '2023-05-01',
            patient: {
              alder: 30,
              kon: 'Kvinna',
              namn: 'Olle',
              id: '123',
            },
          },
        ],
      },
    ],
  })
  store.dispatch(updateShowPersonalInformation(showPersonalInformation))
  renderWithRouter(<PatientHeader patient={patient} />)
}
```